src/avatar_action.cpp

```diff
@@ -94,6 +94,12 @@ bool avatar_action::move( avatar &you, map &m, const tripoint &d )
         }
         return false;
     }
+
+    //If both legs broken without splints and not already on the ground topple over
+    if (you.get_working_leg_count() < 2 && !you.is_lying_down() && !you.worn_with_flag(flag_SPLINT)) {
```

Comment:
With splints on, the player character still shouldn't be able to walk on two broken legs for a while

Reply:
It would appear the game needs crutches (as an available item, that is).

Reply:
I've added CRUTCH flag and check depends on that flag now. I've also added crutches item(as a weapon, not really balanced etc, just for it to be there) and added crutch flag to cane weapon. Push shortly

Reply:
Changes pushed. Now without crutches or cane or w/e (I'm not gonna search items and add flag to all of them :) ) you revert back to prone mode. Otherwise you can stand up(manually) and dont fall down again